app/containers/MessageComposer/components/RecordAudio/RecordAudio.tsx

```diff
@@ -127,12 +127,12 @@ export const RecordAudio = (): ReactElement | null => {
 					<AudioPlayer fileUri={recordingRef.current?.getURI() ?? ''} rid={rid} downloadState='downloaded' />
 				</View>
 				<View style={styles.buttons}>
-					<CancelButton onPress={cancelRecording} />
+					<CancelButton onPress={cancelRecording} accessibilityLabel='Delete_recording' />
```

Comment:
`CancelButton` is only used here.
Adding a new prop for a11y is overcomplicating things.
Add it directly to the component instead.